CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,10 @@
+## Authorino Code of Conduct v1.0
+
+This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to the 3scale "Porta" Component.
+
+- Participants will be tolerant of opposing views.
+- Participants must ensure that their language and actions are free of personal attacks and disparaging personal remarks.
+- When interpreting the words and actions of others, participants should always assume good intentions.
+- Behaviour which can be reasonably considered harassment will not be tolerated.
+
+This Code of Conduct is adapted from the [The Ruby Community Conduct Guideline](https://www.ruby-lang.org/en/conduct/)
```

Comment:
I know it shouldn't matter, but it still feels a bit weird that we have this line in a Golang project.

Reply:
You're 100% right: it doesn't matter.  :)